brage-migration-common/src/main/java/no/sikt/nva/brage/migration/common/model/record/BrageVersion.java

```diff
@@ -0,0 +1,28 @@
+package no.sikt.nva.brage.migration.common.model.record;
+
+import com.fasterxml.jackson.annotation.JsonValue;
+
+public enum BrageVersion {
```

Comment:
Naming: this field is never used by the Brage-migration, only excel scraping. 

Maybe lend something from NVA?
This PublisherAuthority is in NVA a boolean and these values will eventually be scraped to them.

So I suggest PublisherAuthorityEnum?

Reply:
Sounds good. Just copied the name mindlessly from BrageLicence.java